src/SourceBuild/content/repo-projects/Directory.Build.props

```
@@ -38,12 +38,15 @@
     <!-- Update nuget.config property to point to modified file that will be created/updated during build. -->
     <NuGetConfigFile Condition="'$(OriginalNuGetConfigFile)' != ''">$(BaseIntermediateOutputPath)$([System.IO.Path]::GetFileName('$(OriginalNuGetConfigFile)'))</NuGetConfigFile>
 
+    <RepoAssetManifestsDir>$([MSBuild]::NormalizeDirectory('$(BaseIntermediateOutputPath)', 'AssetManifests'))</RepoAssetManifestsDir>
```

The task that will merge these manifests together will need to find all of them, i.e. via this glob pattern: `$(ArtifactsObjDir)*/AssetsManifest/*`. This is a bit fragile as the glob could include more than expected. Can we change this to a common intermediate folder so that it's easier for the task to find all manifests?

```suggestion
    <RepoAssetManifestsDir>$([MSBuild]::NormalizeDirectory('$(ArtifactsObjDir)', 'AssetManifests', '$(RepositoryName)'))</RepoAssetManifestsDir>
```

---

Thanks - will implemented offline, and push the changes after testing.